instrumentation/opentelemetry-instrumentation-pymssql/src/opentelemetry/instrumentation/pymssql/__init__.py

```diff
@@ -0,0 +1,180 @@
+# Copyright The OpenTelemetry Authors
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+"""
+The integration with pymssql supports the `pymssql`_ library and can be enabled
+by using ``PyMSSQLInstrumentor``.
+
+.. _pymssql: https://pypi.org/project/pymssql/
+
+Usage
+-----
+
+.. code:: python
+
+    import pymssql
+    from opentelemetry.instrumentation.pymssql import PyMSSQLInstrumentor
+
+    PyMSSQLInstrumentor().instrument()
+
+    cnx = pymssql.connect(database="MSSQL_Database")
+    cursor = cnx.cursor()
+    cursor.execute("INSERT INTO test (testField) VALUES (123)"
+    cnx.commit()
+    cursor.close()
+    cnx.close()
+
+API
+---
+"""
+from typing import Any, Callable, Collection, Dict, NamedTuple, Tuple
+
+import pymssql
+
+from opentelemetry.instrumentation import dbapi
+from opentelemetry.instrumentation.instrumentor import BaseInstrumentor
+from opentelemetry.instrumentation.pymssql.package import _instruments
+from opentelemetry.instrumentation.pymssql.version import __version__
+
+_DATABASE_SYSTEM = "mssql"
+
+
+class PyMSSQLConnectMethodArgsTuple(NamedTuple):
```

Comment:
Private class?

Reply:
I'm assuming you mean adding an underscore before the class name?

`class _PyMSSQLConnectMethodArgsTuple`?